tests/checkNodeVersion.test.js

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS 
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const { describe, it, expect } = require('@jest/globals')
+const checkNodeVersion = require('../src/checks/checkNodeVersion')
+
+describe(`checks node version ^14 || ^16`, function () {
+    it('returns status: pass', async () => {
```

Comment:
I would add an additional test condition to all tests where the JSON object is `null`.

Reply:
Done.